jmx-scraper/src/main/resources/solr.yaml

```diff
@@ -43,7 +43,7 @@ rules:
       Mean:
         metric: solr.request.time.average
         type: gauge
-        unit: 'ms'
+        unit: "ms"
```

Comment:
I believe we used double quotes only in cases where unit was annotated. Plain curly braces caused parse error.